packages/dialtone-vue2/components/item_layout/item_layout.mdx

```diff
@@ -25,3 +25,20 @@ import * as DtItemLayoutStories from './item_layout.stories.js';
 ```jsx
 import { DtItemLayout } from '@dialpad/dialtone-vue';
 ```
+
+### With default type
+
+By default, item layout includes custom styling, like paddings, sizes, colors, etc.
+
+```html
+<dt-item-layout />
+```
+
+### With custom type
+
+Setting the `type` property to `custom`, removes all the custom styling maintaining the slots positions.
+This way you can utilize the layout and customize your own styling.
```

Comment:
Maybe explain further how can the consumer use their own styling.
For what I see, this component is mostly used as a recipe, so is the idea that the consumer sets the paddings on the inner components? How could the padding for `dt-item-layout` be set? Or is the idea to use a wrapper if you need a padding for the whole thing?

Reply:
`dt-item-layout` is basically just a wrapper component with slots, so If no styling is set on the slots you can pass whatever styling you want.

I think the explanation is enough, or do you have an example of what do you want to add here?

Reply:
hmm does it make more sense to say "spacing" instead of styling? styling could also refer to typography and such.

Reply:
@juliodialpad can you give an example of how can I set the gap to `var(--dt-space-300)`?

Reply:
> hmm does it make more sense to say "spacing" instead of styling? styling could also refer to typography and such.

It also remove some colors on the subtitle slot so it's not just spacing, I this styling its better than spacing in that case.

Reply:
> @juliodialpad can you give an example of how can I set the gap to `var(--dt-space-300)`?

You will not set the gap of the layout, you'll need to use unstyled version of the layout and use margin/padding on the slot, e.g:

```html
<dt-item-layout unstyled>
  <template #left>
      <span class="d-mr8">
        some content
     </span>
    </template>
</dt-item-layout>
```

I can add a prop to the layout to be able to set the gap, but I don't think that's what we want to do with this component.

Reply:
Great, thanks for clarifying that. 
> I can add a prop to the layout to be able to set the gap, but I don't think that's what we want to do with this component.
I don't think so either, maybe when this component is used as a recipe we want to add a gap prop to the parent component.